syntax = "proto3";
package JaegerNet;

import "CreateLobbyMessage.proto";
import "ConnectMessage.proto";

message JaegerNetRequest
{
    oneof MessageType
    {
        CreateLobbyRequest CreateLobbyRequest = 1;
        ConnectRequest ConnectRequest = 2;
    }
}

message JaegerNetResponse
{
    oneof MessageType
    {
        CreateLobbyResponse CreateLobbyResponse = 1;
        ConnectResponse ConnectResponse = 2;
    }
}
